Cargo.toml

```
@@ -21,12 +21,12 @@ include = [
 edition = "2018"
 
 [dependencies]
-embedded-hal = "0.2.6"
+embedded-hal = "1.0.0"
 nb = "1"
 
 [dev-dependencies]
-linux-embedded-hal = "0.3"
-embedded-hal-mock = "0.8"
+linux-embedded-hal = "0.4.0"
+embedded-hal-mock = "0.10.0"
```

```suggestion
embedded-hal-mock = {version="0.10.0", default-features=false, features=["eh1"]}
```
So that we avoid the dependency on `embedded-hal` 0.2